src/AI/mcp/server/handlers/__init__.py

```
@@ -0,0 +1,51 @@
+    ToolSuccess,
+    ToolRecommendation,
+    RouteResult,
+    PrerequisiteError,
+    require_prerequisite,
+    SCHEMA_HINTS,
+    get_tool_registry,
+    get_tool_by_name,
+)
+
+
+def register_tool(name=None, description=None, title=None, annotations=None, meta=None,
```

Ruff (0.15.5)

[warning] Unused function argument: `title`

(ARG001)